packages/admin-ui/src/Button/IconButton.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Button, ButtonProps } from "./Button";
+import { makeDecoratable } from "@webiny/react-composition";
+
+interface IconButtonProps extends ButtonProps {
+    icon: React.ReactNode;
+}
+
+const IconButtonBase = (props: IconButtonProps) => {
```

Comment:
Done, but let me know what you think about the approach I took. We can discuss alternatives too, like some libs use `leftSection` / `rightSection` props instead of mentioning the word `icon`.